web_widget_x2many_2d_matrix/__manifest__.py

```diff
@@ -14,6 +14,7 @@
         "Camptocamp, "
         "CorporateHub, "
         "Onestein, "
+        "SodexisTeam, "
```

Comment:
I don't think the amount of work for this migration qualifies for upgrading to co-authorship. Instead, you should put yourself in contributors.